### PR TITLE
Added the random isogramword selection code

### PR DESCRIPTION
**Pull request by 26pooja**

### CODE REVIEW

File: FBullCowGame.cpp [hunk 1]
```diff
@@ -14,113 +14,182 @@ bool FBullCowGame::IsGameWon() const { return bGameIsWon; }
...
+	//We will add the difficulty level to max tries. This will essentially just increase or
+	//decrease the max number of tries by unity.
+	return (WordLenghtToMaxTries[MyHiddenWord.length()] + static_cast<int>(DifficultyLevel));
 }
 
 void FBullCowGame::Reset()
 {
-    const FString HIDDEN_WORD = "ant"; // this must be an isogram
-    MyHiddenWord = HIDDEN_WORD;
+	//const FString HIDDEN_WORD = "ant"; // this must be an isogram
```
Comment (ufrshubham) [inline FBullCowGame.cpp]: Always remove code if not needed. Don't keep commented code.

File: FBullCowGame.cpp [hunk 2]
```diff
@@ -14,113 +14,182 @@ bool FBullCowGame::IsGameWon() const { return bGameIsWon; }
...
+		{
+			return false;
+		}
+	}
+	return true;
 }
+
+FString FBullCowGame::GetRandomWord(int32 GuessNumber) const
+{
+	FString Randomword;
```
Comment (ufrshubham) [inline FBullCowGame.cpp]: If possible rename this to RandomWord

File: FBullCowGame.cpp (hunk 1 above)
Comment (26pooja) [reply FBullCowGame.cpp]: removed the commented code

File: FBullCowGame.cpp (hunk 2 above)
Comment (26pooja) [reply FBullCowGame.cpp]: changed the name to RandomWord

File: FBullCowGame.cpp [hunk 3]
```diff
@@ -14,113 +14,182 @@ bool FBullCowGame::IsGameWon() const { return bGameIsWon; }
...
+FString FBullCowGame::GenerateRandomWord(FString FileName) const
+{
+	FString String;
+	FStream fs;
+	int32 NoOfLines = 0;
+	int32 random = 0;
+	fs.open(FileName, std::ios::in);
+	if (fs.is_open())
+	{
+		while (!fs.eof())
```
[the remark this answers is not in the document]
Comment (26pooja) [reply FBullCowGame.cpp]: Done

File: FBullCowGame.cpp [hunk 4]
```diff
@@ -14,113 +14,182 @@ bool FBullCowGame::IsGameWon() const { return bGameIsWon; }
...
+			return String;
+		}
+	}
+	if (String.empty())
+	{
+		GenerateRandomWord(FileName);
+	}
+	fs.close();
+
+}
```
[the remark this answers is not in the document]
Comment (26pooja) [reply FBullCowGame.cpp]: Done

File: FBullCowGame.cpp [hunk 5]
```diff
@@ -14,113 +14,182 @@ bool FBullCowGame::IsGameWon() const { return bGameIsWon; }
...
+	random = rand() % NoOfLines;
+	NoOfLines = 0;
+	fs.clear();
+	fs.seekg(0, fs.beg);
+	while (getline(fs, String))
+	{
+		++NoOfLines;
+		if (NoOfLines == random)
+		{
+			return String;
```
[the remark this answers is not in the document]
Comment (26pooja) [reply FBullCowGame.cpp]: Made changes

File: FBullCowGame.cpp [hunk 6]
```diff
@@ -14,113 +14,182 @@ bool FBullCowGame::IsGameWon() const { return bGameIsWon; }
...
+		}
+	}
+	if (String.empty())
```
[the remark this answers is not in the document]
Comment (26pooja) [reply FBullCowGame.cpp]: Was not required so removed

**Comment by ufrshubham** [comment]
I have merged changes from this branch to master. Please delete this branch.

**Pull request merged**